code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -136,6 +136,27 @@
 	taste_description = "rich earthy pungent"
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
 
+/datum/reagent/consumable/nutriment/clothiment
+	name = "Clothiment"
```

Comment:
nit and you dont have to: this would make more sense as something like "cloth fibers" or something

Reply:
As much as I like my silly joke name, cloth fibers is a better name to understand how it works, so I will change it.